src/Sarif.Multitool/RebaseUriOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("rebaseuri", HelpText = "Rebase the URIs in one or more sarif files.")]
+    internal class RebaseUriOptions : MultitoolOptionsBase
+    {
+        [Value(0,
+            MetaName = "<files>",
```

Comment:
>MetaName [](start = 12, length = 8)

we need this argument to be similar to other commands that provide references to files, can we look at that? check out the way the driver framework declares these for the Analyze base command

Reply:
Moved to a MultipleToolFileOptionsBase that handles this properly (with a helper function that works like the driver's).